content/introduction/philosophy/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Gruntwork Philosophy"
+date: 2019-05-31
```

Comment:
Should we always bump the date forwards to a future date?

Reply:
I updated this because it is used in the `Last updated` marker. So we should update the date everytime we modify it so that that marker is accurate.

Reply:
Technically yes, but that's a different point. I was trying to say if you know you're going to publish this doc say later this week, set it to a future date such as `2019-06-05` so it looks 'fresh' when it goes live. WDYT?

Reply:
Ahhh I gotcha. Sorry I misunderstood. Yes that makes sense. We just have to be careful to time it right so that we don't have negative last modified timestamps.